src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Infrastructure/ApiClients/RoatpAssessorApiClient.cs

```diff
@@ -20,9 +20,11 @@ public RoatpAssessorApiClient(string baseUri, ILogger<RoatpAssessorApiClient> lo
             _httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", tokenService.GetToken(_httpClient.BaseAddress));
         }
 
-        public async Task AssignAssessor(Guid applicationId, AssignAssessorApplicationRequest request)
+        public async Task<bool> AssignAssessor(Guid applicationId, AssignAssessorCommand request)
```

Comment:
Not sure about this returning a bool that's true if the StatusCode is ok.  Feels lossy.  what was the reason to do it this way, rather than return the result and the reciever deal with the statuscode?

Reply:
We've used this pattern elsewhere and the caller only need to know if it was successful or not. Exposing HttpStatusCodes beyond the remit of the API Client is a leaky abstraction. Logging and exception handling is already contained in the API Client.

Previously this call was completely lossy, which is okay as fire & forget will work in that scenario, however I think returning true/false is a bit better.

Reply:
That's fine. Will mark as resolved